assets/js/modules/analytics/setup.js

```diff
@@ -656,12 +664,24 @@ class AnalyticsSetup extends Component {
 						</Radio>
 					</Fragment>
 				}
+				{ onSettingsPage && useSnippet && (
+					<div className="googlesitekit-setup-module__input">
+						<Switch
+							id="anonymizeIPAddress"
+							label={ __( 'Anonymize IP address', 'google-site-kit' ) }
+							onClick={ this.switchStatus( 'anonymizeIPAddress' ) }
+							checked={ anonymizeIPAddress }
+							hideLabel={ false }
+						/>
+						<p>{ anonymizeIPAddress ? __( 'IP addresses will be anonymized', 'google-site-kit' ) : __( 'IP addresses will not be anonymized', 'google-site-kit' ) }</p>
```

Comment:
I think we should add a Learn More link here (https://support.google.com/analytics/answer/2763052) similar to how we do for the AMP opt-in, especially since this is enabled by default. People should be informed as to what this does.

![image](https://user-images.githubusercontent.com/1621608/70693802-d91ed400-1cc6-11ea-96c2-ae1d123e8f21.png)

Reply:
Hmm… I find that documentation to be quite technical and not really offer advice or help for less-experienced users.

I go between offering something being better than nothing and thinking that text will just add cognitive load to the page for no reason.

The only people using this feature should be ones already knowing what it is and why they'd want to disable it. I argued for not having the toggle, but I guess at least adding more info as to what it does makes sense.